package.json

```
@@ -33,7 +34,7 @@
     "tape": "^5.3.1"
   },
   "engines": {
-    "node": ">=10.12.0"
+    "node": ">=12.20.0"
```

Would prefer 👇 here, ref: https://github.com/sindresorhus/meta/discussions/15#discussioncomment-688195

```suggestion
    "node": "^12.20.0 || ^14.13.1 || >=16.0.0"
```

---

do we want to avoid the hole: "why isn't v15 supported"? question

---

Both Node.js 13 & 15 is end of life at this point, so I think it's an easy question to answer.

ref: https://nodejs.org/en/about/releases/

Are you suggesting that we stick to `>=12.20.0`? I think it would be good to indicate the minimum version of 14.x as well. Potentially we could do `^12.20.0 || >=14.13.1` but I think that it's better not to support 15.x since it's end-of-life...